crates/dap/src/adapters.rs

```diff
@@ -389,6 +393,54 @@ pub async fn download_adapter_from_github(
     Ok(version_path)
 }
 
+#[cfg(target_os = "windows")]
+async fn extract_zip(zip_path: &Path, target_dir: &Path) -> Result<()> {
```

Comment:
I’m pretty sure we have this function already defined elsewhere in the codebase. Would be good to use the existing version instead of duplicating.